play-json/jvm/src/main/scala/play/api/libs/json/EnvWrites.scala

```diff
@@ -134,14 +141,35 @@ trait EnvWrites {
    * The default typeclass to write a `java.time.Instant`,
    * using '2011-12-03T10:15:30Z' format.
    */
-  implicit val DefaultInstantWrites = new Writes[Instant] {
-    def writes(i: Instant): JsValue = JsString(i.toString)
+  implicit val DefaultInstantWrites: Writes[Instant] =
+    Writes[Instant] { i => JsString(i.toString) }
+
+  /**
+   * The default typeclass to write a `java.time.LocalTime`,
+   * using '10:15:30' format.
+   */
+  implicit val DefaultLocalTimeWrites: Writes[LocalTime] =
+    temporalWrites[LocalTime, DateTimeFormatter](DateTimeFormatter.ISO_TIME)
+
+  /**
+   * Serializer for `java.time.LocalTime` as JSON number.
+   *
+   * {{{
+   * import java.time.LocalTime
+   * import play.api.libs.json.Writes
+   *
+   * implicit val ltnWrites = Writes.LocalTimeNumberWrites
+   * }}}
+   */
+  val LocalTimeNumberWrites: Writes[LocalTime] = Writes[LocalTime] { t =>
```

Comment:
How would you do that? What I mean is you could use `localTimeNumberWrites(TimeUnit.SECONDS)` to write the number of seconds in the day as a number.

Also, this uses nanoseconds by default and the `LocalDateTimeNumberWrites` uses milliseconds since the epoch. If I were to use both together I would expect both to default to the same time unit.